cmd/gen-manifests/main.go

```diff
@@ -589,10 +625,15 @@ func main() {
 					if skipNoconfig {
 						continue
 					}
-					panic(fmt.Sprintf("no configs defined for image type %q", imgTypeName))
+					panic(fmt.Sprintf("no configs defined for image type %q for %s", imgTypeName, distribution.Name()))
 				}
 
 				for _, itConfig := range imgTypeConfigs {
+					if configs.needsSkipping(distribution.Name(), itConfig) {
+						fmt.Printf("Skipping %s for %s/%s\n", itConfig.Name, imgTypeName, distribution.Name())
```

Comment:
Tiniest nitpick: it would be nice if we also logged the reason.

Reply:
Excellent suggestion, thank you! Added.